cli/cmd/project/edit.go

```diff
@@ -71,6 +72,11 @@ func EditCmd(cfg *config.Config) *cobra.Command {
 				req.Public = &public
 			}
 
+			if cmd.Flags().Changed("prod-ttl-seconds") {
+				promptFlagValues = false
+				req.ProdTtlSeconds = &prodTTL
+			}
```

Comment:
Can we make it never prompt for this? If not explicitly set, it should just stay unchanged

Reply:
It is just setting the values here, We won't prompt for this you can see below, we just set it if this flag is `changed`